components/tabs/index.css

```diff
@@ -269,144 +216,107 @@
 .spectrum-Tabs-selectionIndicator {
 	position: absolute;
 	inset-inline-start: 0;
-	background-color: var(--highcontrast-tabs-selection-indicator-color, var(--mod-tabs-selection-indicator-color, var(--spectrum-tabs-selection-indicator-color)));
+	background-color: var(--spectrum-tabs-selection-indicator-color);
 
 	/* Be below the tab */
 	z-index: 0;
 
-	transition: transform var(--mod-tabs-animation-duration, var(--spectrum-tabs-animation-duration)) var(--mod-tabs-animation-ease, var(--spectrum-tabs-animation-ease));
+	/* @todo: set transition: none for initial positioning */
+	transition: transform var(--spectrum-tabs-animation-duration) var(--spectrum-tabs-animation-ease);
 	transform-origin: top left;
 
-	border-radius: var(--mod-tabs-divider-border-radius, var(--spectrum-tabs-divider-border-radius));
+	border-radius: var(--spectrum-tabs-selection-indicator-border-radius);
 }
 
 .spectrum-Tabs--horizontal {
 	align-items: center;
 
 	.spectrum-Tabs-item {
 		vertical-align: top;
-
-		/* Target anything since React likes to add lots of happy <div>s around all things */
-		& + *:not(.spectrum-Tabs-selectionIndicator) {
-			margin-inline-start: var(--mod-tabs-item-horizontal-spacing, var(--spectrum-tabs-item-horizontal-spacing));
-		}
 	}
 
 	.spectrum-Tabs-selectionIndicator {
 		position: absolute;
 		inset-block-end: 0;
-		block-size: var(--mod-tabs-divider-size, var(--spectrum-tabs-divider-size));
+		inline-size: 100%;
+		block-size: var(--spectrum-tabs-selection-indicator-thickness);
+		margin-block-start: var(--spectrum-tabs-label-to-selection-indicator);
 	}
 
 	&.spectrum-Tabs--compact {
-		/* The ActionButton is taller than the tabs, so don't push tabs around */
+		/* The overflow ActionButton is taller than the tabs, so don't push tabs around */
```

Comment:
The [overflow tabs](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/tabs-overflow--compact) in SWC use an action button for the arrows, I'm assuming that's what's meant here.